apps/timetable/admin/index.html

```diff
@@ -0,0 +1,150 @@
+<html>
+    <head>
+        <title>Timetable Administration</title>
+
+        <meta name="viewport" content="width=device-width">
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+
+        <!-- .cam.ac.uk domain Typekit inclusion -->
+        <script type="text/javascript" src="//use.typekit.com/hyb5bko.js"></script>
+        <script type="text/javascript">try{Typekit.load();}catch(e){}</script>
+
+        <link rel="stylesheet" href="/shared/vendor/css/font-awesome.css">
+        <link rel="stylesheet" href="/shared/vendor/css/bootstrap.css">
+        <link rel="stylesheet" href="/shared/vendor/css/chosen.css">
+        <link rel="stylesheet" href="/shared/vendor/css/fullcalendar.css">
+        <link rel="stylesheet" href="/shared/gh/css/gh.base.css">
+        <link rel="stylesheet" href="/shared/gh/css/gh.components.css">
+        <link rel="stylesheet" href="/shared/gh/css/gh.skin.css">
+        <link rel="stylesheet" href="/apps/admin/ui/css/admin.css">
+        <link rel="stylesheet" href="/admin/ui/css/admin.css">
+        <link rel="stylesheet" href="/admin/ui/css/skin.css">
+        <link rel='stylesheet' href="/shared/gh/css/gh.print.css" media="print">
+    </head>
+    <body>
+        <div id="gh-page-container">
+            <div id="gh-left-container">
+                <div id="gh-meta-container">
+                    <a href="http://cam.ac.uk" id="gh-header-logo" target="_blank" title="Visit the Cambridge University website">
+                        <img src="/shared/gh/img/unilogo.png" alt="Cambridge University">
+                    </a>
+                    <div id="gh-content-description">
+                        <p class="gh-academic-year-select">2014 - 2015</p>
+                    </div>
+                </div>
+                <div id="gh-modules-container"><!-- --></div>
+            </div>
+            <div id="gh-right-container">
+                <div id="gh-header-container">
+                    <div id="gh-header"><!-- --></div>
+                    <div id="gh-subheader"><!-- --></div>
+                </div>
+                <main id="gh-main"><!-- --></main>
+            </div>
+            <div class="clearfix"></div>
+        </div>
+
+        <script id="gh-header-template" type="text/template">
+            <h1>Timetable Administration</h1>
+            <% if (gh.data.me) { %>
+                <form id="gh-signout-form" class="form-inline pull-right" role="form">
+                    <button type="submit" class="btn btn-default gh-btn-secondary">
+                        <i class="fa fa-sign-out"></i>Sign out
+                    </button>
+                </form>
+
+            <!--/* TODO: Add else clause that shows a login button (Raven login) */-->
```

Comment:
Thought: if logging in is consistent across the apps then we might be able to consolidate all of that into a partial instead of having logic in our HTML files.